pure_pursuit/src/pure_pursuit_core.cpp

```diff
@@ -52,10 +54,9 @@ PurePursuitNode::~PurePursuitNode()
 void PurePursuitNode::initForROS()
 {
   // ros parameter settings
+  std::string out_twist, out_ctrl_cmd;
   private_nh_.param("velocity_source", velocity_source_, 0);
   private_nh_.param("is_linear_interpolation", is_linear_interpolation_, true);
-  private_nh_.param(
-    "publishes_for_steering_robot", publishes_for_steering_robot_, false);
```

Comment:
While I agree that this name doesn't make sense, the API is in-use by many launch files so removing it completely is problematic. Please re-add this and make it interchangable with the new `output_interface` parameter.

Reply:
Resovled in 7ae4a89a67d0294db0c545114b600c1343bc276d.